### DISCUSSION TITLE
Unable to start second rabbitmq node on single windows 2019 server

### DISCUSSION THREAD

**Discussion by brookhab** [rabbitmq/rabbitmq-server#2732] [Questions] [answered]
I am trying to run two nodes on the same windows server but I was unable to start the second one on `port 5673` . 
The first node is running fine as shown below.
![image](https://user-images.githubusercontent.com/11322420/105243567-d5e65080-5b34-11eb-9b91-ec857b38249b.png)

When I startup the second node it defaults to 0.0.0.0 fails to start . 

Basic information for troubleshooting : 
My configuration: 
`rabbitmq.conf` :

```
listeners.tcp.default = 5673
listeners.tcp.local    = 127.0.0.1:5673
```

` rabbitmq-env-conf.bat` :
```
 set RABBITMQ_CONFIG_FILE=%APPDATA%\RabbitMQ\rabbitmq.conf
 set RABBITMQ_NODENAME=rabbit2@computername
 set RABBITMQ_DIST_PORT=5673
```

Erlang Version : `erl-23.1 `
Rabbit MQ Version : `3.8.9`
Windows version :  `Microsoft Windows Server 2019 Standard`

This is how I am trying to run the server using powershell : 
`PS D:\rabbit\RabbitMQServer2\rabbitmq_server-3.8.9\sbin> .\rabbitmq-server.bat start`

This is the error output I am seeing :
```
Configuring logger redirection

  ##  ##      RabbitMQ 3.8.9
  ##  ##
  ##########  Copyright (c) 2007-2020 VMware, Inc. or its affiliates.
  ######  ##
  ##########  Licensed under the MPL 2.0. Website: https://rabbitmq.com

  Doc guides: https://rabbitmq.com/documentation.html
  Support:    https://rabbitmq.com/contact.html
  Tutorials:  https://rabbitmq.com/getstarted.html
  Monitoring: https://rabbitmq.com/monitoring.html

  Logs: c:/Users/username/AppData/Roaming/RabbitMQ2/log/rabbit2@computername.log
        c:/Users/username/AppData/Roaming/RabbitMQ2/log/rabbit2@computername_upgrade.log

  Config file(s): (none)

  Starting broker...
BOOT FAILED
===========
Error during startup: {error,
                          {could_not_start_listener,"0.0.0.0",5673,
                              eaddrinuse}}

{"init terminating in do_boot",{error,{could_not_start_listener,"0.0.0.0",5673,eaddrinuse}}}
init terminating in do_boot ({error,{could_not_start_listener,0.0.0.0,5673,eaddrinuse}})
```
Can someone point me what I am missing here ?

**Comment by lukebakken** [comment]
Is it correct that the first server has been started as a Windows service?

**Reply by brookhab** [reply]
That is correct, I overlooked that detail. The first server is running as Windows Service. I also want to run the second one as Windows Service after starting the node.

**Comment by lukebakken** [comment, marked as answer]
Please see this discussion:

https://groups.google.com/g/rabbitmq-users/c/VgdYACHQ9vM/m/r0yUFZFNAAAJ
